ambry-api/src/main/java/com.github.ambry/store/StoreStats.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import com.github.ambry.utils.Pair;
+
+
+/**
+ * Exposes important stats related to a {@link Store}.
+ */
+interface StoreStats {
```

Comment:
nope. Store should be considered to be agnostic to these. ServiceId and containerId are our BlobStore concepts and not to be tied to store as such. Think of it as, if someone comes up with another BlobStore impl, would this api makes sense to them